src/sheet.py

```diff
@@ -19,6 +20,7 @@ class Sheet():
     """
     def __init__(self, workbook: Workbook, name: str=None):
         self.workbook = workbook
+        self.sheet = None
```

Comment:
You should be able to remove this as the `if else` block below will always assign `self.sheet`.